javascript/javascript3/week2/Tasks/script.js

```diff
@@ -0,0 +1,41 @@
+
+//Promise that resolves after set time
+function resolveAfterSeconds(resolveAfter){
+    return new Promise((resolve)=>{
+        setTimeout(()=>{
```

Comment:
Code tip:
You could replace this with `setTimeout(resolve, resolveAfter * 1000)` and save yourself a function!